inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -59,6 +62,10 @@ if (ENABLE_TESTS)
     add_subdirectory(src/openvino/test_utils)
 endif()
 
+if(ENABLE_WHEEL)
+    add_subdirectory(install)
```

Comment:
`install`? I think the folder name is incorrect, because it prepares the package but it doesn't install anything.

Reply:
`wheel` folder name sounds more appropriate.

Reply:
setup.py can build other distributions too. For example, you can install python package directly

Reply:
What other distributions setup.py can build? The main purpose of it is to build wheel. Isn't it?

Reply:
Yes, the main scenario for user is building a wheel. However _setup.py_ allows three modes: build, install and develop. All of them should be reflected.

Reply:
Renamed to wheel again)